frontend/src/common/queries/cellGuide.ts

```diff
@@ -52,7 +52,9 @@ async function fetchQuery({
     method: "GET",
     signal,
   });
-  if (response.status === 204) return undefined;
+  if (response.headers.get("Content-Length") === "0") {
```

Comment:
If the content-length is 0, that means S3 returned a 404 error code which [cloudfront now redirects to an empty file called "404"](https://github.com/chanzuckerberg/single-cell-infra/pull/854). This is to prevent the browser from throwing a bunch of red error messages if the requested resource does not exist.

Reply:
let's add this comment in the code, so people remember!